portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.test.js

```diff
@@ -0,0 +1,116 @@
+const { getReportDueDate, getFormattedReportDueDate, getReportPeriodMonthAndYear, getFormattedReportPeriod } = require('./utilisation-report-helpers');
+const externalApi = require('../../../external-api/bank-holidays');
+
+jest.mock('../../../external-api/bank-holidays');
+
+const originalProcessEnv = process.env;
+
+describe('utilisation-report-helpers', () => {
+  beforeEach(() => {
+    process.env = { ...originalProcessEnv };
+    jest.useRealTimers();
+  });
```

Comment:
Does `jest.useFakeTimers()` not need to be reset after it is called (ie. do you not need an `afterAll` function)?